nemo/collections/tts/torch/g2ps.py

```diff
@@ -11,4 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# TODO (xueyang): deprecate this file since no other places import modules from here anymore. However,
+#  all checkpoints uploaded in ngc used this path. So it requires to update all ngc checkpoints g2p path as well.
```

Comment:
@redoctopus @ekmb what is the best way to deprecate `g2ps.py` and `tts_tokenizers.py` as described in the `TODO`?

Reply:
We should make a list of models that are affected and that we'll need to replace on NGC. At that point we can figure out how long it will take to retrain everything and set a release for deprecation.

Reply:
yes, we need to define a strategy to do this and beyond this. model releases usually tie to the latest versioning of tooling. We may need to update models already in NGC in a monthly cadence aligning with NeMo release cadence. Otherwise, NGC models would fall apart because NeMo is moving so fast.

Reply:
Would it be possible to update the model loading logic itself to alias the old paths to the new paths? Rather than physically update the paths in the checkpoints themselves.

Reply:
That's also a legit workaround, but I guess it is not easy to maintain by humans. Retraining would just cost computer time rather than human time.

Reply:
If we want to change the model updating or checkpoint load/restore setup, that may be a good topic for a sync.

A module move of this scale tends to be relatively rare, and models don't tend to break frequently, so a monthly update schedule might lead to some redundant work. But it might be good to have a process for updating models automatically anyway, for when it does happen.